### DISCUSSION TITLE
Unable to see scraped pictures or video

### DISCUSSION THREAD

**Discussion by fadangalini** [andresdelcampo/GameList_Editor#7] [Q&A] [answered]
Hi.
I have connected to my Pi 4 remotely, and can see the systems and games, and can edit the text, change to hidden etc. But I cannot see any of the scraped pictures or videos. I used screenscraper to get all the content originally, and it is all visible when in emulation station. Any ideas why I can't see the pics or videos remotely, I would like to go through fix incorrect or missing items. Many thanks.

**Comment by andresdelcampo** [comment]
Hi fadangalini,

If you are not running the latest version, please update it. There was an issue related to images and video paths that was solved in that same version. 

If it is the latest, I would like to hear more. Could you send me an example of image or video path from the gamelist.xml file that you know it does not show and a description or image of how the images and videos folder structure is? -with respect to the roms location.  
Thanks!

**Reply by andresdelcampo** [reply]
Also curious, if it is the latest version, does it work with the previous one??

**Reply by fadangalini** [reply]
Good Morning.
I am using Gamelist Editor 2022.4
This is the only version I have used, I can install an older version and try if you like? If so, which version do you recommend I try?

Gamelist.xml
The same file seems to be in two locations, but both files are identical and have the edits I made with Gamelist Editor yesterday.
They are here: /opt/retropie/configs/all/emulationstation/gamelists/arcade
and here: /home/pi/.emulationstation/gamelists/arcade

From Gamelist.xml file:
Rom Location
/home/pi/RetroPie/roms/arcade/hypsptsp.zip
Image Location
/home/pi/.emulationstation/downloaded_media/arcade/screenshots/hypsptsp.png
Video Location
/home/pi/.emulationstation/downloaded_media/arcade/videos/hypsptsp.mp4

The laptop is using Windows 10 Home if that’s any help.

If you need more info, let me know.

Thanks for your help

**Reply by andresdelcampo** [reply]
Thanks, that should be enough info for me to investigate the issue over the weekend. 

I suggested trying a previous version (in this case I would recommend 2022.02) in case the path fix included in 2022.04 caused a regression in a scenario like yours. If that were the case, you would get it fully functioning with 2022.02 and you would not need to wait for a fix while I investigate further. That said, I do not know upfront if 2022.02 would work or if the issue you are experiencing is something the tool never handled before. Hopefully it unblocks you right away.

**Reply by fadangalini** [reply]
Hi,

I just tried version 2022.02, but no different sadly. Thanks for looking into this issue.

**Comment by fadangalini** [comment]
Thanks for your quick reply.

I think I have the latest version, but will check that and the rest you've
asked for and get back to you tomorrow. I am at work now till late.

Thanks

**Comment by andresdelcampo** [comment, marked as answer]
I see what's happening now. When you are connecting from Windows to your Emulation station image, you are using the file system that it exposes through \\\\RETROPIE or \\\\192.168... (whichever the the image IP is). Those folders are not the whole Linux filesystem, so an absolute path will never work connecting remotely from Windows. 

I would recommend to move your images and videos closer to your roms so that you can use relative paths. For example, one subdirectory below roms (images and videos subdirectories in the arcade roms directory). So the filesystem could look like this 
```
roms/arcade/hypsptsp.zip
roms/arcade/images/hypsptsp.png
roms/arcade/videos/hypsptsp.mp4
```
Then you can have in the gamelist.xml relative paths like
```
<image>./images/hypsptsp.png</image>
<video>./videos/hypsptsp.mp4</video>
```
which will work both from EmulationStation and connecting remotely from Windows. You can do a Replace All in a text editor in the gamelist.xml so that changing the paths is fast to do. Or even Replace in Files to do all replacements at once. Make a backup first if you think something could go wrong. 

Hope it helps!

**Reply by fadangalini** [reply]
Thanks for looking into this.

I've just given it a try, with a couple of games, didn't want to dive in and change everything to start with.
Still not working sadly.
The videos and images are still working on Emulation station, but not showing on Gamelist editor.

**Reply by andresdelcampo** [reply]
Sorry to hear. That should have worked fine. Note that if you had Gamelist Editor open while moving your games, you need to refresh the system list for it to reload the altered gamelists. As an alternative, to help troubleshoot the problem, I would suggest trying that locally in Windows folders with a couple of games and once that works apply that to your Pi image. It should equally work if it can open the roms list.

**Reply by fadangalini** [reply]
Bit of a pain.... I will have another play later when I have a chance and I'll let you know how I get on.

Thanks
